README.md

```diff
@@ -117,6 +117,7 @@ Flags:
 | **--config** | Path to a [build config](#-build-config)                         |       ❌      |
 | --tls-verify | Require HTTPS and verify certificates when contacting registries |    `true`     |
 | **--type**   | [Image type](#-image-types) to build                             |    `qcow2`    |
+| --rootfs     | Root filesystem type to create. eg. ext4, xfs                    |    `ext4`     |
```

Comment:
Can you at least just document here that this functionality will also be accessible in the future via `bootc install` configs?

Reply:
Isn't that getting ahead of ourselves though? I think it's confusing to refer to future things that don't work right now.